client/src/Components/TaskForm/DateSelector.js

```diff
@@ -0,0 +1,20 @@
+
+import React, { useState } from "react";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+
+import "./DateSelector.css";
+
+const DateSelector = () => {
+	const [startDate, setStartDate] = useState(new Date());
```

Comment:
How does this data get _out_ of the component? The user can edit the local state, but it doesn't seem to be connected to anything else.

Reply:
We will have possible pass it through a function that's passed as prop back to the component where the DateSelector is being used
is that a good option here?

Reply:
Yes, that's a common pattern - just as this component passes the `startDate` and a callback to `setStartDate` to `DatePicker, users of this component would do the same. But until it's actually wired up it shouldn't be included - it's confusing to show something in the UI that doesn't actually _do_ anything.